pom.xml

```diff
@@ -19,16 +19,17 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.kaaproject.kaa</groupId>
     <artifactId>sandbox-frame</artifactId>
-    <version>1.3.2-SNAPSHOT</version>
+    <version>1.4.1-SNAPSHOT</version>
```

Comment:
why 1.4.1? AFAIK we plan to significantly rework KSF, so i believe that 2.0.0 is more suitable according to http://semver.org/